projects/igniteui-angular/src/lib/services/overlay/overlay.ts

```diff
@@ -536,6 +536,7 @@ export class IgxOverlayService implements OnDestroy {
         if (eventArgs.cancel) {
             return;
         }
+        this.removeModalClasses(info);
```

Comment:
Is there a reason why we have a `_hide` and a `hide` methods, when the latter is only a wrapper for the former?

Reply:
P.S. We may also want to update the comment on line `715` - since the method will return if no animations have been assigned or when the animation has started.